core/ledger/util/couchdbtest/couchdbTest.go

```diff
@@ -0,0 +1,45 @@
+/*
```

Comment:
we don't really use camelCase for filenames.  I'd justname the file `couchdb.go`